access/email/app.go

```diff
@@ -293,11 +293,18 @@ func (a *App) onDeletedRequest(ctx context.Context, reqID string) error {
 }
 
 // getEmailRecipients converts suggested reviewers to email recipients
-func (a *App) getEmailRecipients(ctx context.Context, suggestedReviewers []string) []string {
+func (a *App) getEmailRecipients(ctx context.Context, roles []string, suggestedReviewers []string) []string {
 	log := logger.Get(ctx)
-	recipients := stringset.NewWithCap(len(suggestedReviewers) + len(a.conf.Delivery.Recipients))
+	recipients := stringset.New()
 
-	recipients.Add(a.conf.Delivery.Recipients...)
+	for _, role := range roles {
+		roleRecipients := a.conf.RoleToRecipients[role]
+		if len(roleRecipients) == 0 {
+			roleRecipients = a.conf.RoleToRecipients[types.Wildcard]
+		}
```

Comment:
The wildcard recipients usage is handled differently in the Slack plugin - https://github.com/gravitational/teleport-plugins/blob/a532a50d47fdd26704ccd05f19891814301cf10c/access/slack/app.go#L373-L376

I think your interpretation makes more sense and we should definitely keep it consistent. It's probably worth generalizing this function and adding it to `recipients_map.go`. 

```
func GetRecipientsSet(roleToRecipients RoleToRecpients, roles []string, suggestedReviewers []string) []string {
	recipients := stringset.New()

	for _, role := range roles {
		roleRecipients := roleToRecipients[role]
		if len(roleRecipients) == 0 {
			roleRecipients = roleToRecipients[types.Wildcard]
		}

		recipients.Add(roleRecipients...)
	}

	// Suggested reviewers should be validated by caller
	for _, reviewer := range suggestedReviewers {
		recipients.Add(reviewer)
	}

	return recipients.ToSlice()
}
```